### PR TITLE
Add english session access

### PR DESCRIPTION
**Pull request by kitwtnb**
## Overview (Required)
- Get English session
- Divide to Japan or other

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java [hunk 1]
```diff
@@ -24,17 +25,20 @@
 
     private static final int MAX_PER_PAGE = 100;
 
+    private static final String LOCALE_JA = "ja";
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: Can you move this to `LocaleUtil`?

**Comment by konifar** [comment]
@kitwtnb CI was not stable,  @gfx added 2 hacks.
https://github.com/DroidKaigi/conference-app-2017/pull/194
https://github.com/DroidKaigi/conference-app-2017/pull/195

They were already merged.
Can you merge master and push again?

**Comment by konifar** [comment]
@kitwtnb Ping just in case 🙇

**Comment by kitwtnb** [comment]
sorry, I do it later.

**Comment by kitwtnb** [comment]
Sorry, I was late. 🙇 
I finished push, so thank you. 😆

**Comment by konifar** [comment]
LGTM! Thanks for contribution!

**Pull request merged**